pom.xml

```diff
@@ -265,7 +272,7 @@
         <dropwizard.version>1.3.29</dropwizard.version>
         <dropwizard-swagger.version>1.3.17-1</dropwizard-swagger.version>
         <dropwizard.metrics.version>4.1.16</dropwizard.metrics.version>
-        <guava.version>30.0-jre</guava.version>
+        <guava.version>31.1-jre</guava.version>
```

Comment:
would this version change affect alloy?

Reply:
No, I've already successfully build Alloy with engine having this dependency